frontend/components/EmptyTable/_styles.scss

```diff
@@ -3,7 +3,7 @@
     display: flex;
     flex-direction: column;
     align-items: center;
-    margin: 96px auto $pad-large; // 96px to top of div
+    margin: 96px auto; // 96px to top of div
```

Comment:
cleans up vertically off center margins around empty state
Iterate more in future with with ui designer input